### PR TITLE
Remove compiler name subdirectory in prebuilt DALI TF prebuilt directory

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

#### Why we need this PR?
- Refactoring to improve usage of prebuilt DALI TF plugins.

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     *Remove compiler version subdir in prebuilt directory*
     *Removed exact compiler version match requirements from the DALI TF installation script, since TF is compiled with the newer version than the compiler available in the custom op docker.*
 - Affected modules and functionalities:
     *DALI TF package*
 - Key points relevant for the review:
     *NA*
 - Validation and testing:
     *Existing tests apply*
 - Documentation (including examples):
     *NA*

**JIRA TASK**: *[DALI-2160]*

### CODE REVIEW

**Comment by jantonguirao** [comment]
!build

File: dali_tf_plugin/dali_tf_plugin_install_tool.py [hunk 1]
```diff
@@ -54,21 +49,21 @@ def __init__(self, plugin_dest_dir = None):
         #   major version and the minor version in the prebuilt plugin is lower than the requested one.
         self.can_install_prebuilt = not self.always_build and \
             bool(self.tf_compiler) and \
-            self.tf_compiler in self.prebuilt_compilers and \
+            (StrictVersion(self.default_cpp_version) >= StrictVersion('5.0') and \
```
Comment (JanuszL) [inline dali_tf_plugin/dali_tf_plugin_install_tool.py:52]: Why you check the system compiler here? We don't need that to check prebuild. Only the second check is needed (regarding TF compiler version).

**Review by JanuszL — APPROVED** [review]

**Review by klecki — APPROVED** [review]

**Comment by klecki** [comment]
This is not compatible with CI test 310. Please sync with @JanuszL

**Comment by JanuszL** [comment]
> This is not compatible with CI MR 310. Please sync with @JanuszL

When it is merged I adjust.

**Comment by jantonguirao** [comment]
!build

**Pull request merged**